Do not send nocull command to cachefiles

The kernel side of cachefiles does not handle the nocull command.
When cachefilesd sends nocull to cachefiles the kernel module responds
with EOPNOTSUPP and cachefilesd terminates with a configuration error.
Index: cachefilesd-0.10.10/cachefilesd.c
===================================================================
--- cachefilesd-0.10.10.orig/cachefilesd.c
+++ cachefilesd-0.10.10/cachefilesd.c
@@ -433,6 +433,7 @@ int main(int argc, char *argv[])
 		if (memcmp(cp, "nocull", 6) == 0 &&
 		    (!cp[6] || isspace(cp[6]))) {
 			culling_disabled = true;
+			continue;
 		}
 
 		/* note the cull table size command */
